yarn.lock

```diff
@@ -2,46 +2,60 @@
 # yarn lockfile v1
 
 
-"@babel/code-frame@^7.0.0":
```

Comment:
Are these expected to be updated?

Reply:
Yes, since we have added a new dependency `winston`, it is expected that the yarn.lock file is updated to include winston and its dependencies (adding new or updating existing).